pkgs/top-level/all-packages.nix

```diff
@@ -16363,12 +16363,15 @@ with pkgs;
   ffmpeg_4 = callPackage ../development/libraries/ffmpeg/4.nix {
     inherit (darwin.apple_sdk.frameworks) Cocoa CoreMedia VideoToolbox;
   };
+  ffmpeg_5 = callPackage ../development/libraries/ffmpeg/5.nix {
+    inherit (darwin.apple_sdk.frameworks) Cocoa CoreMedia VideoToolbox;
+  };
 
   # Aliases
   ffmpeg_3 = ffmpeg_3_4;
   # Please make sure this is updated to the latest version on the next major
   # update to ffmpeg
-  ffmpeg = ffmpeg_4;
+  ffmpeg = ffmpeg_5;
```

Comment:
this is going to cause mass rebuilds. To just get "ffmpeg_5" into nixpkgs, you could skip the default bump and just expose the new top-level attr.

Reply:
Yup, that's why I targeted staging. I wanted to bump it and see what broke - my laptop isn't powerful enough to rebuild everything.

I'll drop the later two commits and re-send them in another PR